### DISCUSSION TITLE
Question about `notebook.output.wordWrap`

### DISCUSSION THREAD

**Discussion by darkopetrovic** [microsoft/vscode-jupyter#15449] [Questions and Answers]
Is the settings `notebook.output.wordWrap` supposed to word wrap the notebook output cells?

Seems that it does not work for me:

<br /><br />

![image](https://github.com/microsoft/vscode-jupyter/assets/1554288/24fb93b1-21ed-4de0-80a7-d12e223a4607)

**Comment by DonJayamanne** [comment]
@amunger /cc

**Comment by amunger** [comment]
That setting only applies to built-in renderers and not on all mime types - try clicking the output menu to the left of the output, change presentation, and see which renderer you are using.

**Comment by darkopetrovic** [comment]
@amunger Clicking the `...` button on the left does not show the "Change presentation" option. I don't know why.

**Reply by amunger** [reply]
that would imply that there is only one output mime type. 
Looking at your screenshots again, the output actually is wrapped - there is not a horizontal scrollbar. It looks like your inputs are wrapping at a specific character limit, which is not currently supported for outputs

**Reply by darkopetrovic** [reply]
Ok got it now. The output is only wrapped given the window size and not the character limit. 

Thanks.
